src/Stock.js

```diff
@@ -32,32 +33,40 @@ class Stock extends React.Component {
     }
 
     /**
-    * Fetches US Stock Market API information from Alphadvantage, and stores data in stockDayValues and stockPriceValues
+    * Fetches US Stock Market API information from Alphadvantage, and stores data in stockTimeValues and stockPriceValues
     */
 
     getStock() {
         const API_KEY = process.env.API_KEY;
         const selfPointer = this;
-        let stockDayValuesInner = [];
+        let stockTimeValuesInner = [];
         let stockPriceValuesInner = [];
-        let daily_Stock_API_Call = `https://www.alphavantage.co/query?function=TIME_SERIES_DAILY&symbol=${this.state.stockTicker}&apikey=${API_KEY}`
-
-        fetch(daily_Stock_API_Call)
+        let API_Mode = ''
+        let API_Link = ''
+        if (this.state.graphMode === '100 days of ') {
+            API_Link = `https://www.alphavantage.co/query?function=TIME_SERIES_DAILY&symbol=${this.state.stockTicker}&apikey=${API_KEY}`
+            API_Mode = 'Time Series (Daily)'
+        }
+        else {
```

Comment:
instead of doing else here, I think you should do an else if (this.state.graphMode === <the intraday wording>). That just prepares us to add more functionality in the future if we want to, for example, add a 3rd graph type button.

Reply:
Done